runtime-games/renpy/autobuild/build

```diff
@@ -0,0 +1,49 @@
+abinfo "Appending freetype CFLAGS ..."
+export CFLAGS="${CFLAGS} $(pkg-config --cflags freetype2)"
+
+abinfo "Creating build configuration ..."
+cat > "$SRCDIR/renpy/vc_version.py" << EOF
+branch = 'master'
+nightly = False
+official = False
+version = "$PKGVER"
+version_name = 'TBD'
+EOF
+
+abinfo "Building runtime module ..."
+cd "$SRCDIR"/module
+python3 -m build --wheel --no-isolation
+
+abinfo "Building Ren'Py ..."
+cd "$SRCDIR"
+python3 -m compileall 'renpy'
+
+abinfo "Installing Ren'Py ..."
+mkdir -pv "$PKGDIR"/usr/share/renpy
+
+cp -av \
+    "$SRCDIR"/sdk-fonts \
+    "$SRCDIR"/launcher \
+    "$SRCDIR"/renpy \
+    "$SRCDIR"/renpy.py \
+    "$SRCDIR"/the_question \
+    "$SRCDIR"/tutorial \
+    "$SRCDIR"/gui \
+    "$PKGDIR"/usr/share/renpy
+install -Dvm644 "$SRCDIR"/launcher/game/images/logo.png \
+    "$PKGDIR"/usr/share/pixmaps/renpy.png
+install -Dvm644 "$SRCDIR"/sphinx/source/license.rst \
+    "$PKGDIR"/usr/share/doc/renpy/LICENSE.sphinx
+install -Dvm644 "$SRCDIR"/README.rst \
+    "$PKGDIR"/usr/share/doc/renpy/README.rst
+
+install -dvm755 \
+    "$PKGDIR"/usr/share/renpy/lib/py3-linux-$ARCH
```

Comment:
Was this $PYTHON supposed to be Python architecture? If so, it will not match the dpkg architecture.